README.md

```diff
@@ -150,6 +150,31 @@ Please follow the platform configuration ⬇️ before jump to the `Dart` code.
 </array>
 ```
 
+### `ios/Runner/AppDelegate.swift` file changes
+
+- To get a correct callback behvior and no endless Loading on iOS the content of the AppDelegate.swift should be extended.
+
+```AppDelegate.swift
+import Flutter
+import UIKit
+import MSAL
+
+@main
+@objc class AppDelegate: FlutterAppDelegate {
+  override func application(
+    _ application: UIApplication,
+    didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
+  ) -> Bool {
+    GeneratedPluginRegistrant.register(with: self)
+    return super.application(application, didFinishLaunchingWithOptions: launchOptions)
+  }
+
+  override func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool {
```

Comment:
I don't think it is required because example is working completely fine and we are also using it on production app and no issues are there.

Reply:
This is needed only when safari browser / authenticator app is used as auth middleware. we are using webview in one of our production app so didn't notice it. 😃